### PR TITLE
Reorder docs to match each variable for the ARN

### PR DESCRIPTION
**Pull request by deleugpn**
While reading I found it a bit confusing what the layer-version meaning was until I actually found it all the way down to the page. Perhaps it would be nice if the explanation were to follow the order of the variables.

### CODE REVIEW

**Comment by mnapoli** [comment]
That's a good idea thanks!

File: docs/runtimes/README.md [hunk 1]
```diff
@@ -15,10 +15,21 @@ This page is an introduction to the runtimes. The next sections (PHP functions,
...
+### Supported regions (`<region>`)
+
+- `us-east-2`
+- other regions will be supported soon
+
+Remember to use the `<region>` that matches your application, else Lambda will not find the layer.
+
+### Account
+
+Bref Account id is: `416566615250`
```
Comment (mnapoli) [inline docs/runtimes/README.md]: I think having a placeholder + section for that is not necessary, it is simpler without it. Could you re-insert the account ID in the pattern above?

Comment (deleugpn) [reply docs/runtimes/README.md]: Done

**Comment by mnapoli** [comment]
Thanks!

**Pull request merged**